examples/k8s/k8s-controlplane.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2016-2017 IBM Corporation
```

Comment:
please move this file to testing/kubernetes/k8s-controlplane.yaml
and then
in testing/run_tests.sh and testing/build_and_run.sh
comment out the following lines

```bash
if [ -z "$A8_TEST_K8S" ]; then
    A8_TEST_K8S="true"
fi
```

This way, we would only be testing docker examples, until the K8S stuff is complete. However, once the K8S stuff is complete, its a simple matter of uncommenting this code. Everything else is taken care of for you. Including producing the YAML files for the end user.